client/components/Project/AddProjectComponent.vue

```diff
@@ -103,9 +103,35 @@ export default {
     showModal() {
       this.show = true;
     },
-    submit() {
-      console.log(this.fields);
-      this.hideModal();
+    async submit() {
+      const options = {
+        method: 'POST',
+        headers: {'Content-Type': 'application/json'},
+        credentials: 'same-origin',
+        body: JSON.stringify({
+          projectName: this.fields.name,
+          scheduledUpdates: [this.fields.deadline],
```

Comment:
Scheduled updates are optional right? I'm getting a 400 bad request when I try to make a project without a deadline, and it seems like in the frontend it's not expecting the possibility of it either (says reading length of undefined?)

Reply:
Resolved offline!